tests/test_time_series_graph.py

```diff
@@ -550,11 +550,23 @@ def test_from_adjacency_matrices(self):
         matrices[-1][0, 1] = 1
         # there should be 3+1 nodes
         variables = ['X1', 'X2', 'X3']
-        tsdag = TimeSeriesCausalGraph.from_adjacency_matrices(matrices, variables)
-        nodes = sorted(['X1', 'X2', 'X3', 'X1 lag(n=1)'])
-        self.assertEqual(sorted([n.identifier for n in tsdag.nodes]), nodes)
+        tsdag = TimeSeriesCausalGraph.from_adjacency_matrices(matrices, variables).get_minimal_graph()
+        nodes = sorted(['X2', 'X3', 'X1 lag(n=1)'])
```

Comment:
Comment for why you know this is what you want in minimal graph. We know now but will be hard to remember in future.